transmart-batch/docs/data_formats/README.md

```diff
@@ -19,15 +23,15 @@
   * [subject-sample-mapping.md](subject-sample-mapping.md) - Tabular file structure for loading subject sample mappings for HD data.
 * mRNA gene expression data
   * [expression.md](expression.md) - Loading microarray gene expression data.
-  * *under development* - Loading readcounts and normalized readcounts data for mRNAseq and miRNAseq.
+  * *under development* - Loading readcounts and normalized readcounts data for transcripts and miRNA.
```

Comment:
@Spayralbe Something goes wrong here. We now support RNA-Seq transcript and gene level data for readcounts and normalized readcounts indeed.
But what are you referring to with the "*under development* readcounts and normalized readcounts for transcripts and miRNA"? Compared to line 34 on rnaseq.md?

I think the current line meant the miRNAseq (https://wiki.transmartfoundation.org/display/transmartwiki/RC2_test_miRNAseq) and qPCR miRNAseq (https://wiki.transmartfoundation.org/display/transmartwiki/RC2_test_qPCR_miRNA) data types of RC2.

And that you should add the 'transcript' part to line 34.